common_configs/config_log.yaml

```diff
@@ -0,0 +1,44 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+  detailed:
+    format: '%(asctime)s %(pathname)s [%(filename)s:%(lineno)d] [%(process)d] [%(threadName)s] - %(levelname)s - %(message)s'
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
```

Comment:
I think the level for console should be INFO at least, otherwise you will see debug information from viewser lib or HTTP connection.

Reply:
yeah - this was just for debug - ironically